pubspec.yaml

```diff
@@ -11,6 +11,8 @@ authors:
 homepage: https://github.com/Workiva/w_module
 
 dependencies:
+  analyzer: ^0.29.11
```

Comment:
Dependency issues here as well:
>w_module from path depends on analyzer ^0.29.11 and dart_medic >=1.3.6 depends on analyzer ^0.30.0

So, this should be `^0.30.0`, then?

Reply:
```
    conflict: dart_style >=0.2.15 <1.0.2-dev depends on analyzer ^0.29.5
    ! not analyzer ^0.29.5 is satisfied by analyzer ^0.30.0
    ! which is caused by "w_module depends on analyzer ^0.30.0"
    ! thus: dart_style >=0.2.15 <1.0.2-dev is forbidden
    ! dart_style >=0.2.15 <1.0.2-dev is satisfied by dart_style ^0.2.16
    ! which is caused by "w_module depends on dart_style ^0.2.16"
    ! thus: version solving failed
Version solving took 0:00:04.756626 seconds.
Tried 1 solutions.
Because dart_style >=0.2.15 <1.0.2-dev depends on analyzer ^0.29.5 and w_module depends on
  analyzer ^0.30.0, dart_style >=0.2.15 <1.0.2-dev is forbidden.
So, because w_module depends on dart_style ^0.2.16, version solving failed.
```